src/parser/core/modules/Combos.tsx

```diff
@@ -242,6 +244,14 @@ export default class Combos extends Module {
 		return false
 	}
 
+	isAllowableComboBreak(event: AoeEvent, context: AoeEvent[]): boolean {
+		// To be overridden by subclasses. This is called in recordBrokenCombo, and receives the event triggering the broken combo,
```

Comment:
Info should be a TSDoc comment above the function definition so that intellisense and stuff will see it, i.e.

```ts
/**
 * To be overridden...
 */
isAllowableComboBreak(...) {}
```

Reply:
Can do. Want me to change that for the addJobSpecificSuggestions comment while I'm at it?

Reply:
Nvm, I did it anyways :)